src/pages/Admin/Admin.tsx

```diff
@@ -0,0 +1,40 @@
+import Layout from "../../components/Layout/Layout";
+import "./Admin.scss";
+import Header from "../../components/Header/Header";
+import Button from "../../components/Button/Button";
+import placeHolderPFP from "/src/assets/images/placeHolderPFP.svg";
+
+const Admin = () => {
+  const handleSignOut = () => {
+    return;
+  };
+
+  return (
+    <Layout>
+      <Header title={"Admin"} />
+      <div className="admin-page">
+        <div className="admin-page__content">
+          <div className="admin-page__content--photo">
+            <img
+              className="admin-page__content--image"
+              src={placeHolderPFP}
+              alt="your photo"
+            />
+          </div>
+        </div>
+        <div className="add-events-button">
+          <Button label={"Add Events"} onClick={handleSignOut} />
```

Comment:
Can this one navigate to the Page that Luke and Nick are working on?

Reply:
not yet, since we don't have access to what they're working on currently, we can try to pull from their branch and implement the route if necessary

Reply:
Okay that makes sense we can leave this and let them add it in